master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -33,6 +33,14 @@ kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/instal
 
 >[Click here to view the above yaml directly.](1.5/calico.yaml)
 
+## Connecting calicoctl to the Correct etcd
+
+ Calico is not configured to use the etcd run by kubeadm on the Kubernetes master.
+ Instead, it launches its own instance of etcd as a pod, available at
+`http://10.96.232.136:6666`.
+
+Ensure you are connecting to the correct etcd or you will not see any of the expected data.
```

Comment:
This section could just link to the `calicoctl.yaml` docs, right?  Those don't need the user to even think about where etcd is.